src/platforms/x11/graphics/platform.cpp

```diff
@@ -108,9 +108,11 @@ auto mgx::Platform::parse_output_sizes(std::string output_sizes) -> std::vector<
 }
 
 mgx::Platform::Platform(std::shared_ptr<mir::X::X11Resources> const& x11_resources,
+                        std::string const title,
                         std::vector<X11OutputConfig> output_sizes,
                         std::shared_ptr<mg::DisplayReport> const& report)
     : x11_resources{x11_resources},
+      title{title},
```

Comment:
We can move (not copy) title as the parameter is only used to initialize the data member.

```suggestion
                        std::string title,
                        std::vector<X11OutputConfig> output_sizes,
                        std::shared_ptr<mg::DisplayReport> const& report)
    : x11_resources{x11_resources},
      title{std::move(title)},
```